python/lsst/daf/butler/core/_butlerUri/http.py

```diff
@@ -110,15 +110,36 @@ def getHttpSession() -> requests.Session:
                     "export this variable.")
 
     session.verify = ca_bundle
+    log.debug("Session configured and ready.")
+
+    return session
+
+
+def _defaultTimeout() -> float:
+    """Returns the value of the default timeout to use (in seconds).
+
+    Returns
+    -------
+    timeout : `float`
+        Number of seconds to wait before timing out the requests.
+    """
+    timeout = 20
+    return timeout
+
 
+def expect100() -> bool:
```

Comment:
For boolean methods I think I prefer `useExpect100` as the method name so that you are asking a question as to whether expect-100 is required.